src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
```

Comment:
Consider using a loop to create multiple `Ball` objects, as suggested in the checklist: 'Use for loop for creating several objects of the same class'. This will make the code more concise and easier to maintain if the number of balls changes in the future.